src/Module/Xrd.php

```diff
@@ -83,7 +83,7 @@ protected function rawContent(array $request = [])
 		} else {
 			$owner = User::getOwnerDataByNick($name);
 			if (empty($owner)) {
-				DI::logger()->warning('No owner data for user id', ['uri' => $uri, 'name' => $name]);
+				DI::logger()->notice('No owner data for user id', ['uri' => $uri, 'name' => $name]);
```

Comment:
Changed a warning to a notice?

Reply:
Yes. Because it isn't an important event when someones requests data for an unknown account.